### PR TITLE
💄(layout) add main organization logo to course detail aside column

### PR DESCRIPTION
**Pull request by sveetch**
## Purpose

Main organization logo is not displayed yet

## Proposal

This adds the main organization logo at first element into aside
column in course detail using 'show_placeholder' inside a fragment.

### CODE REVIEW

File: src/richie/apps/courses/templates/courses/cms/fragment_organization_main_logo.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{% load cms_tags %}
+{% if current_page.publisher_is_draft %}
+    {% if organization.check_publication is True %}
+        {% show_placeholder "logo" organization.public_extension.extended_object %}
+    {% else %}
+        {{ organization.extended_object.get_title }}
```
Comment (sampaccoud) [inline src/richie/apps/courses/templates/courses/cms/fragment_organization_main_logo.html]: why do you show the title in this case?

File: src/richie/apps/courses/templates/courses/cms/course_detail.html [hunk 2]
```diff
@@ -122,6 +122,12 @@ <h3>{% trans 'License for the content created by course participants' %}</h3>
 
     <div class="course-detail__aside">
 
+      {% if course.organization_main %}
```
Comment (sampaccoud) [inline src/richie/apps/courses/templates/courses/cms/course_detail.html]: main_organization is a required field on the course so you can remove this test.

File: src/richie/apps/courses/templates/courses/cms/fragment_organization_main_logo.html [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{% load cms_tags %}
+{% if current_page.publisher_is_draft %}
+    {% if organization.check_publication is True %}
+        {% show_placeholder "logo" organization.public_extension.extended_object %}
```
Comment (sampaccoud) [inline src/richie/apps/courses/templates/courses/cms/fragment_organization_main_logo.html]: We need to display a default image when the organization has none defined.

File: src/richie/apps/courses/templates/courses/cms/fragment_organization_main_logo.html (hunk 1 above)
Comment (sveetch) [reply src/richie/apps/courses/templates/courses/cms/fragment_organization_main_logo.html]: Forgot to remove this line..

File: src/richie/apps/courses/templates/courses/cms/fragment_organization_main_logo.html (hunk 3 above)
Comment (sveetch) [reply src/richie/apps/courses/templates/courses/cms/fragment_organization_main_logo.html]: But what kind of image we could push when there is none ?

File: src/richie/apps/courses/templates/courses/cms/fragment_organization_main_logo.html [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+        {% show_placeholder "logo" organization.extended_object as logo %}
+    {% endif %}
+{# If the current page is the published version, show only the organizations that are published #}
+{% elif organization.check_publication is True %}
+    {% show_placeholder "logo" organization.public_extension.extended_object as logo %}
+{% endif %}
+{% if logo %}
+    {{ logo }}
+{% else %}
+    <img src="{% static "images/samples/main-organization.png" %}" alt="">
```
Comment (sampaccoud) [inline src/richie/apps/courses/templates/courses/cms/fragment_organization_main_logo.html]: should still have a alt IMO. 
Proposal:
alt="{% trans "organization logo placeholder" %}"

File: src/richie/apps/courses/templates/courses/cms/organization_detail.html [hunk 5]
```diff
@@ -10,7 +10,9 @@
 
   <div class="organization-detail__logo">
     {% with width=216 height=120 %}
-      {% placeholder "logo" %}
+      {% placeholder "logo" or %}
+        <img src="{% static "images/samples/main-organization.png" %}" alt="">
```
Comment (sampaccoud) [inline src/richie/apps/courses/templates/courses/cms/organization_detail.html]: missing "alt"

**Pull request merged**